cmd/rook/object/bucket.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2017 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package object
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+
+	"github.com/rook/rook/cmd/rook/rook"
+	"github.com/rook/rook/pkg/rook/client"
+	"github.com/rook/rook/pkg/util/flags"
+	"github.com/spf13/cobra"
+)
+
+var bucketCmd = &cobra.Command{
+	Use:   "bucket",
+	Short: "Performs commands and operations on object store buckets in the cluster",
+}
+
+func init() {
+	bucketCmd.AddCommand(bucketListCmd)
+	bucketListCmd.RunE = listBucketsEntry
+}
+
+var bucketListCmd = &cobra.Command{
+	Use:   "list",
```

Comment:
why does the rook cli need to list buckets? the overlap feels awkward since the s3 api supports listing buckets